src/js/pages/export/ExportPage.js

```diff
@@ -31,7 +33,19 @@ export class ExportPage extends Component {
     this.props.resetSelected();
   }
 
-  toggleCheck = () => { this.setState({checked: !this.state.checked});}
+  toggleCheck = () => { 
+    this.setState({
+      checked: !this.state.checked,
+      checkedAll: false
+    });
+  }

```

Comment:
when toggling states, this seems to be the accepted way to do it in react
``` this.setState( prevState => ({checked: !prevState.checked, checkedAll: false})```

Reply:
@natibekele oh that looks nice, I'll update the code